Backoffice/DisplayBlock/Strategies/ContentListStrategy.php

```diff
@@ -26,9 +26,9 @@ public function show(ReadBlockInterface $block)
             'class' => $block->getClass(),
             'contentNodeId' => $block->getAttribute('contentNodeId'),
             'characterNumber' => $block->getAttribute('characterNumber'),
-            'keywords' => $block->getAttribute('keywords'),
-            'choiceType' => $block->getAttribute('choiceType'),
-            'contentType' => $block->getAttribute('contentType'),
+            'keywords' => $block->getAttribute('contentSearch')['keywords'],
```

Comment:
@itkg-nanne : You must check that $block->getAttribute('contentSearch') return an array + that each key exists